app/public/js/service_worker_script.js

```diff
@@ -0,0 +1,50 @@
+var url= window.location.href;
+var pos = url.search('using_pwa');
```

Comment:
Move this check to the server.  HomeController.php has a method that would be called for route "/".  Store the true/false value of whether the request was using_pwa into the server's session so it is available as the user navigates around the site.

This value on the server can be used to prevent including these service worker scripts when not in a PWA.